ghost/core/core/server/services/mega/mega.js

```diff
@@ -87,9 +88,6 @@ const sendTestEmail = async (postModel, toEmails, memberSegment) => {
     let emailData = await getEmailData(postModel);
     emailData.subject = `[Test] ${emailData.subject}`;
 
-    if (memberSegment) {
-        emailData = postEmailSerializer.renderEmailForSegment(emailData, memberSegment);
-    }
```

Comment:
Removed as this was redundant here, we do serialization for email segment again in `bulkEmailService.send` before sending out the email.